embassy-lora/src/stm32wl/mod.rs

```diff
@@ -198,77 +140,109 @@ impl<'a> StateInner<'a> {
             .set_preamble_len(8)
             .set_header_type(HeaderType::Variable)
             .set_payload_len(0xFF)
-            .set_crc_en(true)
+            .set_crc_en(false)
```

Comment:
Any particular reason for disabling CRC?

Reply:
I tried to match the C driver from the STM32 SDK but also found this table in RP002-1.0.3:
![grafik](https://user-images.githubusercontent.com/1181037/183478029-1a41db9f-e451-484a-94fe-d8e2270f9f07.png)